Documentation/compatibility/wpf-accessibility-improvements.MD

```diff
@@ -0,0 +1,101 @@
+## 195: Accessibility improvements in WPF
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+__High Contrast improvements__
+
+- Focus for expander was not visible.  This is now fixed.
+- The text in CheckBox and RadioButton is difficult to see when selected.  This issue is now fixed.
```

Comment:
Checkbox and RadioButton should be links:
`T:System.Windows.Controls.CheckBox`
`T:System.Windows.Controls.RadioButton`
in Checkbox and RadioButton --> in the Checkbox and RadioButton controls